### ISSUE TITLE
setting logging.json=true does not truly have full json logs

### ISSUE THREAD

**Issue by vchan2002** [elastic/beats#6064]
When I set logging.json=true, it seems that the logs that was output to STDOUT is not 100% json... there are some additional fields that's there ... I am using heartbeat 6.1.1 in a docker container, and the logs look like this...  There are some additional fields that's there before the JSON part starts.

2018/01/12 05:40:44.257192 manager.go:110: {"level":"INFO","message":"Select (active) monitor http","timestamp":"2018-01-12T05:40:44Z"}

When I run logstash in docker with its logs output setup as JSON, it's fully 100% json, which I was able to have them processed with... logstash to have it sliced and diced .... 

While I can do an additional logstash logic to parse this.... it would be great if there's some sort of standard that we should expect from elastic products when I specify the logs to be in JSON format.

**Comment by andrewkroh** [comment]
This will be fixed in 6.2. #5901

**Comment by GMartinez-Sisti** [comment]
@andrewkroh looks like this is broken again in 6.5.0:

I'm using the following configurations for logging to the console:

```yaml
logging.metrics.enabled: true
logging.to_files: false
logging.metrics.enabled: true
logging.level: info
logging.json: true
```

Message example:

```bash
2018-11-30T18:45:00.517Z	INFO	[monitoring]	log/log.go:144	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":110,"time":{"ms":8}},"total":{"ticks":340,"time":{"ms":36},"value":340},"user":{"ticks":230,"time":{"ms":28}}},"handles":{"limit":{"hard":1048576,"soft":1048576},"open":11},"info":{"ephemeral_id":"87b2219a-e749-4747-bc4f-7cfa4650011a","uptime":{"ms":210013}},"memstats":{"gc_next":5544848,"memory_alloc":4042160,"memory_total":31287632}},"filebeat":{"events":{"added":26,"done":26},"harvester":{"open_files":4,"running":4}},"libbeat":{"config":{"module":{"running":0},"reloads":3},"output":{"events":{"acked":26,"batches":6,"total":26},"read":{"bytes":2175},"write":{"bytes":63116}},"pipeline":{"clients":11,"events":{"active":0,"published":26,"total":26},"queue":{"acked":26}}},"registrar":{"states":{"current":29,"update":26},"writes":{"success":6,"total":6}},"system":{"load":{"1":1.17,"15":1.16,"5":1.13,"norm":{"1":0.2925,"15":0.29,"5":0.2825}}}}}}
```
